gf-guide/metrics.md

```diff
@@ -248,26 +248,85 @@ If the font is already hosted on [fonts.google.com](http://fonts.google.com/), y
 
--   <https://github.com/source-foundry/font-line/issues/2>
--   [SourceHanSansReadMe.pdf](https://github.com/adobe-fonts/source-han-sans/raw/release/SourceHanSansReadMe.pdf)
+**Note** If the font is being built with `makeotf`, be aware that Adobe\'s default settings expect `sTypoMetrics` to match the emBox. Unless glyph heights are manually set for all glyphs, the `vmtx` values will not be set properly. As such, it may be easier to set vertical metrics as usual in the source, and adjust in post-production.
+
+These metrics were established based on investigations into improving metrics performance of CJK fonts across the library. Please see the following issue for more information:
+-   <https://github.com/google/fonts/issues/8911>
+
+### Base Table
+
+In addition to the above metrics, CJK fonts are now required to include a `BASE` table (https://learn.microsoft.com/en-us/typography/opentype/spec/baselinetags) to ensure broad compatibility. 
+
+For a standard square emBox font (1000 units wide for a 1000 UPM font), such as Iansui, the following BASE table was added to the `.fea` file. 
+```
+table BASE {
+  HorizAxis.BaseTagList                 icfb  icft  ideo   romn;
+  HorizAxis.BaseScriptList  DFLT  ideo   -67   827  -120      0,
+                            hani  ideo   -67   827  -120      0,
+                            kana  ideo   -67   827  -120      0,
+                            latn  romn   -67   827  -120      0,
+                            cyrl  romn   -67   827  -120      0,
+                            grek  romn   -67   827  -120      0,
+
+  VertAxis.BaseTagList                  icfb  icft  ideo  romn;
+  VertAxis.BaseScriptList   DFLT  ideo    53   947     0   120,
+                            hani  ideo    53   947     0   120,
+                            kana  ideo    53   947     0   120,
+                            latn  romn    53   947     0   120,
+                            cyrl  romn    53   947     0   120,
+                            grek  romn    53   947     0   120,
+} BASE;
+```
+The BASE tags above are:
+
+- `icfb`: Ideographic character face bottom edge (in HorizAxis) / left edge (in VertAxis)
+- `icft`: Ideographic character face top edge (in HorizAxis) / right edge (in VertAxis)
+- `ideo`: ideographic emBox bottom edge (in HorizAxis) / left edge (in VertAxis)
+- `romn`: Latin baseline. Usually 0 in HorizAxis, inverse of ideo in VertAxis
+
+Some additional tags which may be useful can be reviewed on the [official documentation](https://learn.microsoft.com/en-us/typography/opentype/spec/baselinetags). 
+
+In the case of a design that is narrower than the usual square emBox, it is recommended to also include `idtp` to mark the top edge of the emBox (opposite of `ideo`). For example, [WD XL Lubrifont](https://github.com/NightFurySL2001/WD-XL-font), which has an advance width of 765, has the following base table:
```

Comment:
"In the case of a design that is narrower than the usual square emBox" -> "In the case of a design that is not using the usual square em-box"

Reply:
Also another font for reference can be the [Dela Gothic One](https://fonts.google.com/specimen/Dela+Gothic+One?lang=ja_Jpan), which has a wider design.